src/controls/SelectTeamChannelPicker/ISelectTeamChannelPickerState.ts

```diff
@@ -0,0 +1,5 @@
+import { ITag } from "office-ui-fabric-react";
```

Comment:
Please, use specific imports, otherwise you're referencing the whole OUIFR and increase the size of our lib.

Reply:
fixed